packages/ra-supabase-ui-materialui/src/guessers/ListGuesser.tsx

```diff
@@ -0,0 +1,217 @@
+            )
+            .map((source: string) =>
+                inferElementFromType({
+                    name: source,
+                    types: listFieldTypes,
+                    description:
+                        resourceDefinition.properties![source].description,
+                    format: resourceDefinition.properties![source].format,
+                    type: (resourceDefinition.properties &&
+                    resourceDefinition.properties[source] &&
+                    typeof resourceDefinition.properties[source].type ===
+                        'string'
+                        ? resourceDefinition.properties![source].type
+                        : 'string') as string,
+                })
+            );
+        const inferredTable = new InferredElement(
+            listFieldTypes.table,
+            null,
+            inferredFields
+        );
+        setChild(inferredTable.getElement());
+
+        const rowFilters =
+            schema!
+                .paths![`/${resource}`].get!.parameters?.filter(obj =>
+                    obj['$ref'].includes('rowFilter')
+                )
+                .map(obj => obj['$ref'].split('.').pop()) ?? [];
+        const inferredInputsForFilters = rowFilters
+            .filter(
+                source =>
+                    resourceDefinition.properties![source].format !== 'tsvector'
+            )
+            .map(source => {
+                const field = resourceDefinition.properties![source];
+                return inferElementFromType({
+                    name: source,
+                    types: editFieldTypes,
+                    description: field.description,
+                    format: field.format,
+                    type: field.type as string,
+                });
+            });
+        if (
+            rowFilters.some(
+                source =>
+                    resourceDefinition.properties![source].format === 'tsvector'
+            )
+        ) {
+            const fullTextSearchSource = rowFilters.find(
+                source =>
+                    resourceDefinition.properties![source].format === 'tsvector'
+            );
+            const field = resourceDefinition.properties![fullTextSearchSource!];
+            inferredInputsForFilters.push(
+                inferElementFromType({
+                    name: `${fullTextSearchSource!}@fts`,
+                    types: {
+                        string: {
+                            component: SearchInput,
+                            representation: props =>
+                                `<SearchInput alwaysOn source="${props.source}" />`,
+                        },
+                    },
+                    description: field.description,
+                    format: 'tsvector',
+                    props: { alwaysOn: true },
+                    type: field.type as string,
+                })
+            );
```

Comment:
nitpick (non-blocking): it's a bit disturbing to have the SearchInput as the last filter instead of the first one

![2024-11-07_12-34](https://github.com/user-attachments/assets/439b2004-9bd7-4eec-babc-207afd1aabbf)